src/main/java/com/github/onlinebookstore/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -86,25 +78,13 @@ private void addNewItemToCart(
     }
 
     @Override
-    public ShoppingCartResponseDto updateItem(
-            CreateCartItemRequestDto requestDto,
-            Long userId,
-            Long cartId) {
-        cartItemRepository.findById(cartId).orElseThrow(
-                () -> new EntityNotFoundException("No cart with id " + cartId + " found")
-        );
-
-        Long bookId = requestDto.getBookId();
-        Book book = bookRepository.findById(bookId)
-                .orElseThrow(
-                        () -> new EntityNotFoundException("No book with id " + bookId + " found")
-                );
+    public ShoppingCartResponseDto updateItem(CreateCartItemRequestDto requestDto,
```

Comment:
In the task description you should provide request that has quantity only. It looks like CreateCartItemRequestDto  isn't appropriate